src/mpu9250_interface.hpp

```diff
@@ -21,6 +21,9 @@ class MPU9250Interface {
     const int8_t MPUAddr;
     hwlib::i2c_bus_bit_banged_scl_sda i2c;
 
+    float gyroRes = 250.0f / 32768.0f;
+    float accelRes = 2.0f / 32768.0f;
+

```

Comment:
Add a comment for the magic numbers. And if there is a good reason to use floats add that reason in the comments. Else change them to int's.